homeassistant/components/pi_hole/sensor.py

```diff
@@ -18,8 +19,71 @@
     DATA_KEY_API,
     DATA_KEY_COORDINATOR,
     DOMAIN as PIHOLE_DOMAIN,
-    SENSOR_TYPES,
-    PiHoleSensorEntityDescription,
+)
+
+
+@dataclass
+class PiHoleSensorEntityDescription(SensorEntityDescription):
+    """Describes PiHole sensor entity."""
+
+    icon: str = "mdi:pi-hole"
```

Comment:
Side note: This default icon seems to always be overridden in the descriptions below.

Reply:
nice catch, thanks 🙂 -> #85371